lit_gpt/config.py

```diff
@@ -480,6 +480,23 @@ def norm_class(self) -> Type:
 # NousResearch Hermes
 ######################
 nous_research = [
+    # https://huggingface.co/NousResearch/Nous-Hermes-llama-2-7b/blob/main/config.json
+    dict(
+        org="NousResearch",
+        name="Nous-Hermes-llama-2-7b",
```

Comment:
I tested Nous-Hermes-Llama2-13 and it's working. I tried to test this checkpoint but I'm facing #371. After that's fixed, we can go ahead and merge this

Reply:
I had to change the vocab size. Now it works. Thanks!